packages/schema/package.json

```diff
@@ -22,29 +22,56 @@
     "./breadboard.schema.json": "./breadboard.schema.json",
     "./graph.js": {
       "import": {
-        "types": "./dist/graph.d.ts",
-        "default": "./dist/graph.js"
+        "types": "./dist/src/graph.d.ts",
+        "default": "./dist/src/graph.js"
       }
     }
   },
   "scripts": {
     "build": "wireit",
     "test": "wireit",
+    "test:js": "wireit",
+    "test:ts": "wireit",
+    "test:watch": "wireit",
     "prepack": "wireit",
     "build:tsc": "wireit",
-    "build:update-json-schema": "wireit",
-    "test:validate-json-schema": "wireit"
+    "build:schema": "wireit"
   },
   "wireit": {
     "build": {
       "dependencies": [
         "build:tsc",
-        "build:update-json-schema"
+        "build:schema"
       ]
     },
-    "test": {
+    "test:js": {
+      "command": "node --test-reporter=spec --test ./dist/__tests__/**/*.test.js",
+      "dependencies": [
+        "build"
+      ],
+      "files": [
+        "breadboard.schema.json",
+        "package.json",
+        "dist/**/*",
```

Comment:
Don't think we want to include the dist/ input files here, nor the *.ts files. The `build` dependency is sufficient for that.